<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfiguracionCasas</class>
 <widget class="QDialog" name="ConfiguracionCasas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>429</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="add" stdset="0">
   <string/>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>29</y>
     <width>651</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="Layout">
    <item>
     <widget class="QTableWidget" name="Tabla">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>180</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Casa</string>
       </property>
       <property name="font">
        <font>
         <strikeout>false</strikeout>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">Dispositivo Asociado</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="Layout_Incorporar">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Casa a Incorporar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="casa_nueva"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Dispositivo sin Asociar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ComboDispositivos"/>
      </item>
      <item>
       <widget class="QPushButton" name="boton_agregar">
        <property name="text">
         <string>Agregar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="Layout_Eliminar">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Casa a Eliminar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ComboCasas"/>
      </item>
      <item>
       <widget class="QPushButton" name="Eliminar">
        <property name="text">
         <string>Eliminar</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="boton_aceptar">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>350</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>boton_aceptar</sender>
   <signal>clicked()</signal>
   <receiver>ConfiguracionCasas</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>626</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
